Frontend/voitheia-App/src/app/components/about/about-us/about-us.component.html

```diff
@@ -1 +1,84 @@
-<p>about-us works!</p>
+<title>W3.CSS Template</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<link rel="stylesheet" href="https://www.w3schools.com/lib/w3-theme-black.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<body id="myPage">
+
+  <!-- Team Container -->
+  <div class="w3-container w3-padding-64 w3-center" id="team">
+    <h1>Wer wir sind</h1>
+    <p>Wir sind ein Team aus 9 Entwicklern, Bastlern, kreativen Köpfen und enthusiastischen Weltverbesserern.</p>
+
+    <!--Reihe1-->
+    <div class="w3-row">
+      <br>
+      <div class="w3-quarter">
+        <img src="/assets/images/Team_Lukas.jpg" alt="Lukas" style="width:45%" class="w3-circle w3-hover-opacity">
+        <h3>Lukas</h3>
+        <p>...der im Hintergrund die Fäden zieht</p>
+        <h6> Backend-Entwicklung <br /> Data Security</h6>
+      </div>
+
+      <div class="w3-quarter">
+        <img src="/assets/images/Team_Andi.jpg" alt="Andi" style="width:45%" class="w3-circle w3-hover-opacity">
+        <h3>Andi</h3>
+        <p>...der am hinteren Ende alles rettet</p>
+        <h6> Backend-Entwicklung</h6>
+      </div>
+
+      <div class="w3-quarter">
+        <img src="/assets/images/Team_Sebastian.jpg" alt="Sebastian" style="width:45%" class="w3-circle w3-hover-opacity">
+        <h3>Sebastian</h3>
+        <p>...der immer sicherstellt, dass wir auf Wolke 7 schweben</p>
+        <h6> Cloud-Anbindung </h6>
+      </div>
+
+      <div class="w3-quarter">
+        <img src="/assets/images/Team_Benedikt.jpg" alt="Benedikt" style="width:45%" class="w3-circle w3-hover-opacity">
+        <h3>Benedikt</h3>
+        <p>...der das Ganze nutzbar macht </p>
+        <h6> Frontend-Entwicklung</h6>
+      </div>
+
+    </div>
+
+    <!--Reihe 2-->
+    <div class="w3-row">
+      <br>
+      <div class="w3-quarter">
+        <img src="/assets/images/Team_Kevin.jpg" alt="Kevin" style="width:45%" class="w3-circle w3-hover-opacity">
+        <h3>Kevin</h3>
+        <p>...der Allrounder <br /></p>
+        <h6> Backend-Entwicklung <br />Frontend-Entwicklung <br /> Design </h6>
+      </div>
+
+      <div class="w3-quarter">
+        <img src="/assets/images/Team_Christoph.jpg" alt="Christoph" style="width:45%" class="w3-circle w3-hover-opacity">
+        <h3>Christoph</h3>
+        <p>...der uns mobil macht <br/>  </p>
+        <h6> App Entwicklung</h6>
+      </div>
+
+      <div class="w3-quarter">
+        <img src="/assets/images/Team_Helen.jpg" alt="Helen" style="width:45%" class="w3-circle w3-hover-opacity">
```

Comment:
The placeholder image should only be present once. We should not copy it several times. Including them several times will reduce the speed of loading the page.

Reply:
Ihr schickt mir einfach schnellstmöglich eure Bilder und dann hat sich das auch erledigt :)